jpegutils.h

```diff
@@ -11,50 +11,8 @@
 #ifndef __JPEGUTILS_H__
 #define __JPEGUTILS_H__
 
- /*
-  * jpeg_data:       buffer with input / output jpeg
-  * len:             Length of jpeg buffer
-  * itype:           Y4M_ILACE_NONE: Not interlaced
-  *                  Y4M_ILACE_TOP_FIRST: Interlaced, top-field-first
-  *                  Y4M_ILACE_BOTTOM_FIRST: Interlaced, bottom-field-first
-  * ctype            Chroma format for decompression.
-  *                  Currently always 420 and hence ignored.
-  * raw0             buffer with input / output raw Y channel
-  * raw1             buffer with input / output raw U/Cb channel
-  * raw2             buffer with input / output raw V/Cr channel
-  * width            width of Y channel (width of U/V is width/2)
-  * height           height of Y channel (height of U/V is height/2)
-  */
+int jpgutl_decode_jpeg (unsigned char *jpeg_data, int len,
+                     unsigned int width_in, unsigned int height_in, unsigned char *img);
```

Comment:
It probably would've been nice to keep/update the documentation here.

Or perhaps would've been even nicer to put it in one of the standard formats that editors can use to give contextual quick help, eg.

```
/**
 * Decode a jpeg into ????
 *
 * @param jpeg_data Input data
 * @param len Length of jpeg_data in bytes
etc
 * @return -1 on error, 0 on success
 */
```
(width_in potentially confuses me; I guess it can't be the width of the input jpeg data as that's something that fixed in the jpeg data. I guess it must be the width of the output image.)

I think there's probably quite a strong argument that motion should start to use a struct that wraps up the associated unsigned char */width/height parameters we use a lot into a struct called 'BitmapData' or something to try and make some of the code easier to follow and safer!)... but I digress.